From 9a2c427f52ce66c6e8a09a748ddfbd87d2b4567d Mon Sep 17 00:00:00 2001
From: tarsin <43900799+asuka-mio@users.noreply.github.com>
Date: Sun, 10 Apr 2022 10:06:49 +0800
Subject: [PATCH 2/3] bionic: Turn to mimalloc

---
 libc/Android.bp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libc/Android.bp b/libc/Android.bp
index 8151ada3e..90c9e89f4 100644
--- a/libc/Android.bp
+++ b/libc/Android.bp
@@ -175,7 +175,7 @@ cc_defaults {
         "libc_jemalloc_wrapper",
     ],
     header_libs: ["gwp_asan_headers"],
-    product_variables: libc_scudo_product_variables,
+    product_variables: libc_mimalloc_product_variables,
 }
 
 // Functions not implemented by jemalloc directly, or that need to
-- 
2.37.3

