From 83b266ffdd45d9cf51d8f15842afe96e54fa7db4 Mon Sep 17 00:00:00 2001
From: tarsin <yuanqingxiang233@163.com>
Date: Sun, 18 Sep 2022 00:05:33 +0800
Subject: [PATCH 3/3] bionic: Don't explicitly set scudo in common parts

---
 libc/Android.bp | 18 ++++++------------
 1 file changed, 6 insertions(+), 12 deletions(-)

diff --git a/libc/Android.bp b/libc/Android.bp
index 90c9e89f4..e434df98f 100644
--- a/libc/Android.bp
+++ b/libc/Android.bp
@@ -119,24 +119,18 @@ cc_defaults {
     // warning since this is intended right now.
     ldflags: ["-Wl,-z,muldefs"],
 
-    product_variables: {
-        malloc_zero_contents: {
-            cflags: ["-DSCUDO_ZERO_CONTENTS"],
-        },
-        malloc_pattern_fill_contents: {
-            cflags: ["-DSCUDO_PATTERN_FILL_CONTENTS"],
-        },
-        malloc_not_svelte: {
-            cflags: ["-DUSE_SCUDO"],
-        },
-    },
-
     lto: {
         never: true,
     },
 }
 
 libc_scudo_product_variables = {
+    malloc_zero_contents: {
+            cflags: ["-DSCUDO_ZERO_CONTENTS"],
+        },
+        malloc_pattern_fill_contents: {
+            cflags: ["-DSCUDO_PATTERN_FILL_CONTENTS"],
+        },
     malloc_not_svelte: {
         cflags: ["-DUSE_SCUDO"],
         whole_static_libs: ["libscudo"],
-- 
2.37.3

